C7/Game.cs

```diff
@@ -108,7 +108,13 @@ public override void _Ready()
 		catch(Exception ex) {
 			errorOnLoad = true;
 			PopupOverlay popupOverlay = GetNode<PopupOverlay>(PopupOverlay.NodePath);
-			popupOverlay.ShowPopup(new ErrorMessage(ex.Message), PopupOverlay.PopupCategory.Advisor);
+			string message = ex.Message;
+			string[] stack = ex.StackTrace.Split("\r\n");	//for some reason it is returned with \r\n in the string as one line.  let's make it readable!
+			foreach (string line in stack) {
+				message = message + "\r\n" + line;
+			}
+
+			popupOverlay.ShowPopup(new ErrorMessage(message), PopupOverlay.PopupCategory.Advisor);
```

Comment:
This gives a more useful error message if a file fails to load for some reason.  Yeah, it's in the logs too (which is nice!), but having a hint in-your-face is nice too.  Eventually we probably want to roll this back but when it's mostly developers playing, I think it makes sense.